test-engineering/load-tests/README.md

```diff
@@ -75,3 +78,42 @@ kubectl scale deployment/locust-worker --replicas=20
 Currently, it's set to run `10` workers but it can be changed by running above command.
 
 
+# How to grab the logs
+
+Get the master pod name by running:
+
+```text
+kubectl get pods -o wide
+```
+
+then run:
+
+```text
+kubectl cp <master-pod-name>:/home/locust/merino_stats.csv merino_stats.csv
+```
+
+```text
+kubectl cp <master-pod-name>:/home/locust/merino_exceptions.csv merino_exceptions.csv
+```
+
+```text
+kubectl cp <master-pod-name>:/home/locust/merino_failures.csv merino_failures.csv
+```
+
+To remove all the GET requests and retain only the aggregate summary use:
+
+```text
+cat merino_stats.csv | grep -Ev "^GET," > merino_summary.csv
```

Comment:
We'd need to test this, but another option is seeing if we just overwrite the merino_stats.csv file with the edited version, instead of adding a 4th or 5th file (well, I guess fifth on the **master** node, fourth on the local machine since we don't copy all .csv files from the pod).
But I'm not 100% certain we can <kbd>cat</kbd> a file and pipe it and then redirect it back to itself. It doesn't always work for me and sometimes you're just left with an empty file.

Reply:
I'm sure that we **cannot** cat a file and the redirect it back to itself. The shell truncates the file before executing the command. You could use a temporary file for this

```suggestion
cat merino_stats.csv | grep -Ev "^GET," > merino_stats.csv.tmp
mv merino_stats.csv.tmp merino_stats.csv
```